lib/primitiveValues/bigint.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const constants = require('../constants');
+const formatUtils = require('../formatUtils');
+const lineBuilder = require('../lineBuilder');
+
+const DEEP_EQUAL = constants.DEEP_EQUAL;
+const UNEQUAL = constants.UNEQUAL;
+
+function describe(value) {
+  return new BigIntValue(value);
+}
+exports.describe = describe;
+
+exports.deserialize = describe;
+
+const tag = Symbol('BigIntValue');
+exports.tag = tag;
+
+class BigIntValue {
+  constructor(value) {
+    this.value = value;
+  }
+
+  compare(expected) {
+    return expected.tag === tag && Object.is(this.value, expected.value)
+      ? DEEP_EQUAL
+      : UNEQUAL;
+  }
+
+  formatDeep(theme) {
+    const string = String(this.value);
```

Comment:
I think we should suffix this with `n` for clarity.

Reply:
Agreed that the formatting should show the `n` suffix. Is my latest change a reasonable way to do it, or should I be using the `formatUtils` somehow?